tests/handlers/test_register.py

```diff
@@ -510,6 +519,17 @@ def test_auto_create_auto_join_room_preset_invalid_permissions(self):
         self.assertEqual(rooms, set())
         self.assertEqual(invited_rooms, [])
 
+    @override_config(
+        {
+            "user_consent": {
+                "block_events_error": "Error",
+                "require_at_registration": True,
```

Comment:
This somewhat changes what we're testing by requiring it at registration, which is not the default. 😢